src/logic/linkGithubIdentity.js

```diff
@@ -0,0 +1,37 @@
+import {createLogic} from 'redux-logic';
+
+import {
+  linkGithub,
+  saveCredentialForCurrentUser,
+} from '../clients/firebase';
+import {
+  accountMigrationNeeded,
+  identityLinked,
+  linkIdentityFailed,
+} from '../actions/user';
+import {getProfileForAuthenticatedUser} from '../clients/github';
+import {bugsnagClient} from '../util/bugsnag';
+
+export default createLogic({
+  type: 'LINK_GITHUB_IDENTITY',
+  async process() {
+    try {
+      const {user: userData, credential} = await linkGithub();
+      await saveCredentialForCurrentUser(credential);
+      return identityLinked(userData, credential);
+    } catch (e) {
+      switch (e.code) {
```

Comment:
I know the saga did the same thing (my fault!), but I think it’s a little silly to use a `switch` statement with only one case and a default. Let’s just use an `if`/`else`?